Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -970,63 +999,59 @@ - (void)addAttachmentUploadViewIfNecessary:(UIView *)attachmentView
 
     if (self.isOutgoing) {
         if (!self.attachmentStream.isUploaded) {
-            self.attachmentUploadView = [[AttachmentUploadView alloc] initWithAttachment:self.attachmentStream
-                                                                               superview:attachmentView
-                                                                 attachmentStateCallback:attachmentStateCallback];
+            __unused AttachmentUploadView *attachmentUploadView =
+                //            self.attachmentUploadView =
+                // This view will be added to attachmentView which will retain a strong reference to it.
+                [[AttachmentUploadView alloc] initWithAttachment:self.attachmentStream
```

Comment:
TIOLI: Delete this comment explaining unconventional behavior and replace the method call with self explanatory code like: `[AttachmentUploadView addToView:attachment:attachmentStateCallback:]`

Reply:
I went one better and pulled out the logic to "add" the attachment upload view from its constructor.  That's vestigial.